.kokoro/tests/run_tests.sh

```diff
@@ -118,6 +118,9 @@ ${HOME}/cloud_sql_proxy -instances="${SQLSERVER_INSTANCE}"=tcp:1433 &>> \
        ${HOME}/cloud_sql_proxy-sqlserver.log &
 echo -e "\Cloud SQL proxy started for SQL Server."
 
+# For run/idp example, a Firebase IDP token
+export IDP_KEY=$(gcloud secrets versions access latest --secret="python-docs-samples-idp-key" --project="${GOOGLE_CLOUD_PROJECT}")
```

Comment:
Hi Katie! Would you mind trying out the instructions https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#secrets to add this secret to CI?

Reply:
Sorry @glasnt for leading you astray!

Reply:
No worries, @averikitsch. I shall figure this out eventually!

Reply:
Just confirming that I added the secret for @glasnt just now! :)